tests/scsi/010

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2025 Google LLC
+
+. tests/scsi/rc
+. common/null_blk
+. common/scsi_debug
+
+DESCRIPTION="test F2FS data lifetime support"
+QUICK=1
+
+requires() {
+	_have_fio
+	_have_driver f2fs
+	# See also commit 7643f3fe2772 ("f2fs: assign the write hint per stream
+	# by default"; v6.10).
+	_have_kver 6 10
+	_have_program mkfs.f2fs
+	_have_scsi_debug_group_number_stats
+}
+
+test() {
+	set -e
```

Comment:
This test case will be the second test case to use set -e (errexit) feature. When an error happens between the set -e and set +e, scsi_debug device could be left in mounted status, and could affect to the following test cases. I suggest to factor out the region between set -e and set +e as a function, as you did for zbd/013.

Reply:
This has been fixed.